server/tests/test_parameterval.py

```diff
@@ -16,7 +16,13 @@ def createTestWorkflow(self):
             id_name='stringparam',
             module_version=self.module_version,
             type= ParameterSpec.STRING,
-            def_value='foo')
+            def_value='foo',
+            placeholder='placeholder')
```

Comment:
Later tests don't actually check that the placeholder text is set correctly, but then again it's not reasonable to test that ParameterSpec.objects.create() works correctly as Django tests already do that. So is this placeholder here unnecessary since we never check it? Probably not, as there could conceivably be an edge case where something goes wrong if the placeholder is set.

Reply:
Oh wait, I get it -- this is the base class for the view tests as well.